learn/chapter1/8-camera.md

```diff
@@ -258,12 +265,12 @@ protected override void OnUpdateFrame(FrameEventArgs e)
 {
     //Keyboard movement...
 
-    if (firstMove)
+    if (FirstMove)
     {
         lastPos = new Vector2(mouse.X, mouse.Y);
         firstMove = false;
     }
-    else{    
+    else {
```

Comment:
This code does not follow the OpenTK style guide. Please fix this in the whole article:
```cs
if (statement)
{
    // stuff
}
else
{
    // other stuff. notice that the curly brace always has its own line.
}
```